core/src/main/java/apoc/trigger/Trigger.java

```diff
@@ -18,41 +20,40 @@
  */
 
 public class Trigger {
-    public static class TriggerInfo {
-        public String name;
-        public String query;
-        public Map<String,Object> selector;
-        public Map<String, Object> params;
-        public boolean installed;
-        public boolean paused;
-
-        public TriggerInfo(String name, String query, Map<String, Object> selector, boolean installed, boolean paused) {
-            this.name = name;
-            this.query = query;
-            this.selector = selector;
-            this.installed = installed;
-            this.paused = paused;
-        }
+    // public for testing purpose
+    public static final String DB_NON_WRITER_ERROR = """
+            This instance is not allowed to write to the database because it is a follower.
+            Please use these procedures against the database leader.
+            """;
+    
+    @Context
+    public GraphDatabaseAPI db;
 
-        public TriggerInfo( String name, String query, Map<String,Object> selector, Map<String,Object> params, boolean installed, boolean paused )
-        {
-            this.name = name;
-            this.query = query;
-            this.selector = selector;
-            this.params = params;
-            this.installed = installed;
-            this.paused = paused;
-        }
-    }
+    @Context public TriggerHandler triggerHandler;
 
-    @Context public GraphDatabaseService db;
+    @Context public Log log;
 
-    @Context public TriggerHandler triggerHandler;
+    private void preprocessDeprecatedProcedures() {
+        final String msgDeprecation = """
+                Please note that the current procedure is deprecated,
+                it's recommended to use the `apoc.trigger.install`, `apoc.trigger.drop`, `apoc.trigger.dropAll`, `apoc.trigger.stop`, and `apoc.trigger.start` procedures,
+                instead of, respectively, `apoc.trigger.add`, `apoc.trigger.remove`, `apoc.trigger.removeAll`, `apoc.trigger.pause`, and `apoc.trigger.resume`.""";
```

Comment:
```suggestion
                Please note that the current procedure is deprecated.
                It is recommended to use the `apoc.trigger.install`, `apoc.trigger.drop`, `apoc.trigger.dropAll`, `apoc.trigger.stop`, and `apoc.trigger.start` procedures,
                instead of, respectively, `apoc.trigger.add`, `apoc.trigger.remove`, `apoc.trigger.removeAll`, `apoc.trigger.pause`, and `apoc.trigger.resume`.""";
```

Reply:
Generally speaking in formal wording you would want to use the short-forms:
- isn't
- it's
- doesn't
- mustn't

And instead you should wish to favour long-forms
- is not
- it is
- does not
- must not

Unfortunately I'm not sure how much we adhere to this when writing our documentation and error warnings 🤷